src/test/resources/cucumber/features/integrationTests/sca/scanResultsProcessing.feature

```diff
@@ -53,3 +53,15 @@ Feature: Cx-Flow SCA Integration permutation tests
       |               | 0.0   | 8                        |
       | low           | 0.0   | 0                        |
       |               | -0.3  | 8                        |
+
+
+  @SCA_Issues_Creation
+  Scenario Outline: Publish SCA results and check ADO tickets are getting created
+    Given scan initiator is SCA
+    And bug tracker is Azure
+    When publishing new known unfiltered SCA results with 8 findings including 2 high and 6 medium vulnerabilities
+    Then new <numberOfTickets> tickets should be created
+    Examples:
```

Comment:
"Examples" is not needed here since we only have 1.

Reply:
Right...